Sources/SwerverCore/Router/Router.swift

```diff
@@ -23,17 +25,17 @@ public class Router {
             }
             return responseHeaderFormatter.format(httpResponse: action.dispatch())
         }
-        return "HTTP/1.1"
+        return responseHeaderFormatter.format(httpResponse: notFoundAction.dispatch())
     }
-    
+
     private func fetchMethod(request: HttpRequest) -> [RequestMethod:Action] {
         return routes[request.getUrl()]!
     }
-    
+
     private func routeExist(request: HttpRequest) -> Bool {
         return routes[request.getUrl()] != nil
     }
-    
+
     private func populateRoutes() {
         routes[URL(string: "/demo")!] = [RequestMethod.head: headAction, RequestMethod.get: getAction]
```

Comment:
Why does URL require forced unwrapping here?

Reply:
like the above, if the string provided to create the URL is invalid then it'll return nil thus the forced unwrapping here.